src/content/app/genome-browser/state/browser-bookmarks/browserBookmarksSelectors.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { getBrowserActiveGenomeId } from '../browser-general/browserGeneralSelectors';
+import { defaultBrowserBookmarksStateForGenome } from './browserBookmarksSlice';
+
+import type { RootState } from 'src/store';
+
+export const getActiveGenomeBrowserBookmarks = (state: RootState) => {
+  const activeGenomeId = getBrowserActiveGenomeId(state);
+  const activeGenomeBrowserBookmarks =
+    activeGenomeId && state.browser.browserBookmarks[activeGenomeId];
+
+  return activeGenomeBrowserBookmarks || defaultBrowserBookmarksStateForGenome;
+};
+
+export const getActiveGenomeBookmarks = (state: RootState) => {
```

Comment:
The name of this function looks confusingly similar to `getActiveGenomeBrowserBookmarks`.

Perhaps there is no need for `getActiveGenomeBrowserBookmarks` at all? It doesn't seem to be doing much.

```ts
export const getActiveGenomeBookmarks = (state: RootState) => {
  const activeGenomeId = getBrowserActiveGenomeId(state);
  const noBookmarks = [];
  return activeGenomeId
    ? state.browser.browserBookmarks[activeGenomeId].bookmarks ?? noBookmarks
    : noBookmarks;
}
```

Reply:
✅